UnitySDK/Assets/ML-Agents/Editor/Tests/EditModeTestInternalBrainTensorGenerator.cs

```diff
@@ -1,36 +1,60 @@
 using System.Collections.Generic;
-using System.Linq;
 using Barracuda;
 using NUnit.Framework;
 using UnityEngine;
 using MLAgents.InferenceBrain;
+using System.Reflection;
+
 
 namespace MLAgents.Tests
 {
     public class EditModeTestInternalBrainTensorGenerator
     {
-        static IEnumerable<Agent> GetFakeAgentInfos()
+        static IEnumerable<Agent> GetFakeAgents()
```

Comment:
Rename since it returns Agents. This changed a fair amount in order to initialize the agents before returning